pkg/term/proxy.go

```diff
@@ -0,0 +1,81 @@
+package term
```

Comment:
could we re-use `github.com/docker/docker/pkg/term` here?

Reply:
It's a good idea for pouch to depend on docker's package? I have no idea about it.

Reply:
but the content is from moby's package, right?